.github/dependabot.yml

```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
-- package-ecosystem: bundler
+- package-ecosystem: npm
   directory: "/"
   schedule:
     interval: daily
```

Comment:
Might want to schedule monthly instead, as new npm dependencies are much more frequent.

Reply:
done